packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -20,20 +20,32 @@ import { convertContextualMenuItemToDrawerMenuItem } from '../CallWithChatCompos
 /* @conditional-compile-remove(one-to-n-calling) */
 import { CallCompositeStrings } from '../CallComposite';
 import { AddPeopleButton } from './AddPeopleButton';
+/* @conditional-compile-remove(PSTN-calls) */
+import { CommunicationIdentifier } from '../../../../../common/temp/node_modules/.pnpm/@azure/communication-signaling@1.0.0-beta.13/node_modules/@azure/communication-signaling/types/src';
+/* @conditional-compile-remove(PSTN-calls) */
+import { AddPhoneNumberOptions } from '@azure/communication-calling';
 
 /**
  * @private
  */
 export const PeoplePaneContent = (props: {
   inviteLink?: string;
   onRemoveParticipant: (participantId: string) => void;
+  /* @conditional-compile-remove(PSTN-calls) */
+  onAddParticipant: (participant: CommunicationIdentifier, options?: AddPhoneNumberOptions) => void;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
   strings: CallWithChatCompositeStrings | /* @conditional-compile-remove(one-to-n-calling) */ CallCompositeStrings;
   setDrawerMenuItems: (_DrawerMenuItemProps) => void;
   mobileView?: boolean;
+  /* @conditional-compile-remove(PSTN-calls) */
+  alternateCallerId?: string;
 }): JSX.Element => {
   const { inviteLink, onFetchParticipantMenuItems, setDrawerMenuItems, strings, onRemoveParticipant } = props;
+  /* @conditional-compile-remove(PSTN-calls) */
+  const { alternateCallerId } = props;
```

Comment:
let's use prop directly below instead of declaring conditional attributes her.

Reply:
you can also CC the prop from within the props object as well for a smoother transition to stable. something  like:
`{ ..., onRemoveParticipant, /* @conditional-compile-remove(PSTN-calls) */ alternateCallerId }